documentation/book/topic-operator.adoc

```diff
@@ -235,7 +235,21 @@ Default `6`.
 – The level for printing logging messages. 
 The value can be set to: `ERROR`, `WARNING`, `INFO`, `DEBUG` and `TRACE`. 
 Default `INFO`.
-
+* `STRIMZI_TLS_ENABLED`
+– For enabling the TLS support so encrypting the communication with Kafka brokers and Zookeeper nodes.
```

Comment:
Does this enable encryption with Zoo? I think its only Kafka, or? (since Zoo is handled by the sidecar)

Reply:
Yeah you are right :-(